api/forms.py

```diff
@@ -32,8 +32,17 @@ class CreatePropertyForm(forms.Form):
 	startPrice = forms.DecimalField(initial=600)
 	dateStart = forms.DateTimeField(initial=datetime.now())
 	dateEnd = forms.DateTimeField(initial=datetime.now())
+	availStart = forms.DateTimeField(initial=datetime.now())
+	availEnd = forms.DateTimeField(initial=datetime.now())
```

Comment:
Not sure if you want to initialize the availEnd a few days after today. (3 days from now, or something)